operator/src/main/resources/example-keycloak.yml

```diff
@@ -5,12 +5,18 @@ metadata:
 spec:
   instances: 1
   serverConfiguration:
-    KC_DB: postgres
-    KC_DB_URL_HOST: postgres-db
-#    KC_DB_USERNAME: ${secret:keycloak-db-secret:username}
-#    KC_DB_PASSWORD: ${secret:keycloak-db-secret:password}
-    KC_DB_USERNAME: postgres
-    KC_DB_PASSWORD: testpassword
+    KC_DB:
+      value: postgres
+    KC_DB_URL_HOST:
+      value: postgres-db
+    KC_DB_USERNAME:
+      secret:
+        name: keycloak-db-secret
+        key: username
+    KC_DB_PASSWORD:
+      secret:
```

Comment:
Please keep the encoding consistent with Kubernetes standard using `name` and `value`.
On these lines this difference shows pretty evidently.

Reply:
👍